models/index.go

```diff
@@ -19,7 +19,7 @@ func NewIndexModel(s *myradio.Session) *IndexModel {
 //
 // On success, it returns the current and next show, and nil.
 // Otherwise, it returns undefined data and the error causing failure.
-func (m *IndexModel) Get() (currentAndNext *myradio.CurrentAndNext, banners []myradio.Banner, teams []myradio.Team, err error) {
+func (m *IndexModel) Get() (currentAndNext *myradio.CurrentAndNext, banners []myradio.Banner, teams []myradio.Team, podcasts []myradio.Podcast, err error) {
```

Comment:
Not an issue as such, but is anyone keeping count of the number of requests this model is making? Worth keeping an eye on..

(Also the return tuple is getting ugly - change to object?)